actioncable/blade.yml

```diff
@@ -5,3 +5,22 @@ load_paths:
 
 logical_paths:
   - test.js
+
+plugins:
+  sauce_labs:
+    browsers:
+      Google Chrome:
+        os: Mac, Windows
+        version: -2
+      Firefox:
+        os: Mac, Windows
+        version: -2
+      Safari:
+        platform: Mac
+        version: -3
+      Internet Explorer:
+        version: -1
```

Comment:
I'd add `Microsoft Edge` to the list. Trix has a fairly fresh matrix you can steal: https://github.com/basecamp/trix/blob/master/.blade.yml#L24-L40